### DISCUSSION TITLE
Initial Setup

### DISCUSSION THREAD

**Discussion by duaneellissd** [Motion-Project/motion#1689] [General]
Installed on UBUNTU today:
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.2 LTS"

Installed USB WEB CAM:
Bus 001 Device 005: ID 046d:0826 Logitech, Inc. HD Webcam C525

this is a HOME linux box, so I enabled streaming from any device.

At the main web page, I get a box with a small blue box and a ? mark in the middle.
I can try things like the action menu selections and I see things in the log

For example:
```
[1:ml1] [NTC] [ALL] [Jul 08 20:18:54] motion_detected: Motion detected - starting event 1
[1:ml1] [NTC] [ALL] [Jul 08 20:19:06] mlp_actions: End of event 1
```
This tells me the MOTION program is functional (sort of)

I get this output for other commands:
```
bash$ sudo v4l2-ctl --list-devices
HD Webcam C525 (usb-0000:00:14.0-4):
	/dev/video0
	/dev/video1
	/dev/media0
```

If I run on the Linux Console - I get a black screen with white overlay
I think I need to adjust the camera settings by default it is not seeing anything but a black square using Firefox on linux.
But it otherwise works, or goes through the motions as if it is working.

**Comment by Mr-Dave** [comment]
Have a look at these [videos](https://www.youtube.com/channel/UCZpOjUMnc-uO3qIyZAD_4lg/videos).  They were created using an old version so depending upon the version you are using of motion, the parameter names may have changed.  These used a network camera so you'll need to use a parameter for v4l2 devices.  Type `motion -h` to see the version number and then open the guide that is specific to that version on the [documentation](https://motion-project.github.io/motion_guide.html) page.  NOTE: If the version you have is NOT 4.5, go down on the page to find the correct link.

Also, the title indicates Safari(Mac) but the text you describe has nothing to do with a Mac.  Can you adjust the title accordingly?
